video-presentation.py

```diff
@@ -230,6 +233,43 @@ def parsevalue(size, max_size):
         self.logger.debug("Masks after preparing: %s", parsed_masks)
         return parsed_masks
 
+    def create_video_previews(self, filename):
+        """Create mp4 and webm heavily compressed previews of the presentation to use in the previewer"""
+
+        # Let's not be greedy, use half available cores since we are probably in a docker container
+        encoding_threads = multiprocessing.cpu_count()
+        if encoding_threads > 1:
+            encoding_threads = int(np.ceil(encoding_threads/2))
+
+        ffmpeg_stub = "ffmpeg -loglevel error -y -i \"" + os.path.abspath(filename) + "\" -threads " + \
+                      str(encoding_threads)
+        # We use the same audio settings for both videos
+        no_audio = " -an "
+        audio = " -acodec libvorbis -b:a 64k "
+        # Use very heavy compression since most of what we deal with is 2d without shadows
+        mp4_settings = " -vcodec libx264 -preset medium -b:v 96k -qmax 42 -maxrate 250k "
+        webm_settings = " -vcodec libvpx -quality good -b:v 96k -crf 10 -qmin 0 -qmax 42 -maxrate 250k -bufsize 1000k "
```

Comment:
I'm personally more a fan of creating one command string and filling it with a dict (like we do in EB), but this will work. 

We should check if everything required is installed in the docker container.

Reply:
I've run this and tested it, because opencv is a dep already ffmpeg was already there. The default from the OS restricts the available codecs but there's enough there to suit our purposes.